.github/workflows/continuous_deployment.yml

```diff
@@ -53,14 +55,23 @@ jobs:
         changelogger notes $VERSION --no-pretty >> $GITHUB_ENV
         echo ${DELIMITER} >> $GITHUB_ENV
 
+    - name: Create Archive
+      # Create zip archive for release to composer repository (Packagist)
+      run: |
+        wrapper_dir="magento2-klaviyo"
+        rsync -rc --exclude-from=".distignore" "$GITHUB_WORKSPACE/" $wrapper_dir/ --delete --delete-excluded
+        zip -r ${{ env.ZIP_FILE_NAME }} ./$wrapper_dir
+
     - name: Create GitHub Tag & Release
+      # Creates GH tag and release, adds zip archive as artifact to release.
       uses: ncipollo/release-action@v1
       with:
         tag: ${{ needs.determine-if-deploy.outputs.version }}
         commit: ${{ github.sha }}
         name: Release ${{ needs.determine-if-deploy.outputs.version }}
         body: ${{ env.content }}
         prerelease: true
```

Comment:
Is there a reason we create the release as a pre-release?

Reply:
Synced with Claire offline on this. This needs to be a prerelease so it doesn't prematurely get picked up by packagist until it passes review by Adobe/M2.